pkgs/servers/dns/pdns-recursor/default.nix

```diff
@@ -21,8 +21,11 @@ stdenv.mkDerivation rec {
   configureFlags = [
     "--enable-reproducible"
     "--enable-systemd"
+    "sysconfdir=/etc/powerdns-recursor"
```

Comment:
Why not just `/etc`, to align with upstream default.

Reply:
I went with subdirectories rather than storing the files directly in /etc to more closely follow how NixOS works currently; also, it just felt cleaner to me somehow. If you feel strongly about following upstream defaults, though, I can change it to just use `/etc`

Reply:
@NickCao is this a hard blocker for you? Otherwise I would like getting this merged.

Reply:
Partly, especially that we've (properly) renamed `powerdns` to `pdns` now, so at least `sysconfdir=/etc/powerdns` should be `sysconfdir=/etc/pdns`.

Reply:
@thequux can you update it so we can get this merged?

Reply:
See the latest commit :-)

Reply:
There are still some leftovers in sysconfdir=.